### DISCUSSION TITLE
The software problem with Windows XP

### DISCUSSION THREAD

**Discussion by Michahel** [xiaoyifang/goldendict-ng#445] [Q&A]
Why can't I run the program in Windows XP? I get an error message: "I:\Program Files\GoldenDict\GoldenDict.exe is not a valid win32 application"

**Comment by shenlebantongying** [comment]
Sadly, Qt5.15's minimum version support is Windows 7

Chromium (the thing under Qt-webengine) abandons XP at 2016
